torch_xla/test/test_xla_sharding_base.py

```diff
@@ -0,0 +1,41 @@
+import unittest
+import numpy as np
+
+import torch
+from torch import nn
+import torch_xla
+import torch_xla.core.xla_model as xm
+import torch_xla.experimental.xla_sharding as xs
+from torch_xla.experimental.pjrt import using_pjrt
+
+
+@unittest.skipIf(not using_pjrt() or xm.get_xla_supported_devices("GPU"),
+                 f"Requires PJRT_DEVICE set to `TPU` or `CPU`.")
+class XlaShardingTest(unittest.TestCase):
```

Comment:
I think it is better to put this file under `/test/spmd` instead of `torch_xla/test`

Reply:
As far as I can tell, we need to put it under `torch_xla` so that it can be imported by tests (like `test_util.py` in `torch_xla/test`. I could create `torch_xla/test/spmd` to make it more explicit.

Reply:
I can you put import from python file at the same path, check https://github.com/pytorch/xla/blob/master/test/args_parse.py

Reply:
Done